src/components/notebookRenderStrategy.tsx

```diff
@@ -121,4 +142,20 @@ export class NotebookRenderStrategy implements ExpandableNodeRenderStrategy {
 	private onChevronClick() {
 		this.expandNode();
 	}
+
+	private onExpand() {
+		if (this.notebook.needsToFetchChildren && this.notebook.apiUrl && this.globals.oneNoteDataProvider && !!this.globals.callbacks.onNotebookInfoReturned) {
+			this.globals.oneNoteDataProvider.getNotebookBySelfUrl(this.notebook.apiUrl, 5).then((notebook) => {
+				this.notebook.sections = notebook.sections
+				this.notebook.sectionGroups = notebook.sectionGroups
+			}).catch((apiError: OneNoteApi.RequestError) => {
+				this.notebook.apiHttpErrorCode = apiError.statusCode;
```

Comment:
The catch is an any, not a OneNoteApi.RequestError... that said, this codebase does this multiple times, so I don't think you should change this. I do think that it would be a lot better to get the api error message if available, instead of getting an error message from the response code

Reply:
Fixed for this and sharedNotebooks